app/src/main/java/org/mozilla/fenix/tabstray/syncedtabs/SyncedTabsComposables.kt

```diff
@@ -0,0 +1,321 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Let's drop the `Composables` suffix in the file name. Looking at the sample Compose apps, this is non-standard naming.